htlcswitch/switch.go

```diff
@@ -229,10 +232,21 @@ type Switch struct {
 	// to the forwarding log.
 	fwdEventMtx         sync.Mutex
 	pendingFwdingEvents []channeldb.ForwardingEvent
+
+	// bestHeight is the best known height of the main chain. The links will
+	// use this information to govern decisions based on HTLC timeouts.
+	bestHeight uint32
```

Comment:
Comment should note that this must be used atomically.

Reply:
Also this should be moved to the top of the struct in order to not run into alignment issues with 32-bit systems.

Reply:
Fixed.